.changes/include-image-macro.md

```diff
@@ -0,0 +1,10 @@
+---
+"tauri": "patch:feat"
+"tauri-utils": "patch:feat"
+"tauri-macros": "patch:feat"
+"tauri-codegen": "patch:feat"
```

Comment:
tauri-codegen should have a different change file because it exports a different function `include_image_codegen`